apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -82,6 +82,7 @@ export class TaskCardUc {
 		const taskParams = {
 			name: params.title,
 			courseId: '',
+			private: false,
```

Comment:
I don't understand why we do this combination... 
If we have no course, then the task is always set to private, right?
For the moment, we are not saving the courseId here, which should be changed because we are saving it in the task-card. And then, I would link the logic of the private based on course.

Reply:
We for now had the courseId optional so that way it is added a few lines below. With the change for having the couse required (what was discussed yesterday) that will change so we will always have a course with the task card.
As for now there is no course in the update, simply because the course will not get updated for now and we should implement that once we need it. That is the same with the visibility, as this select is disabled for now as well.
I dont like seting the value for visibility depending on the existence of a course. We want to make that value being set individually in the FE, so that value should be reflected separately in the BE in my opinion.

Reply:
I just tried, there is also the option to choose a course, make the task available and afterwards still set private to "true" so the student can not see the task anymore. setting private individually should be possible so that we can set the visibility this way. making the task visible in the first place requirs visibleAt, course as well as this private setting, so this should be reflected somehow.